.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Can you remove this?

Reply:
🤷 Sure. Though it's useful to have in there for those of us who use JetBrains IDEs, so as to not include those files in the repository.